### PR TITLE
feat(codspeed.io): Try out benchmarking cloud service

### PR DESCRIPTION
**Pull request by janpio**

### CODE REVIEW

**Review by aqrln — APPROVED** [review]

File: packages/client/package.json [hunk 1]
```diff
@@ -149,6 +149,7 @@
     }
   },
   "dependencies": {
+    "@codspeed/benchmark.js-plugin": "^1.0.1",
```
Comment (aqrln) [inline packages/client/package.json:152]: This must be a `devDependency`

**Review by aqrln — CHANGES_REQUESTED** [review]

File: packages/client/package.json [hunk 2]
```diff
@@ -149,6 +149,7 @@
     }
   },
   "dependencies": {
+    "@codspeed/benchmark.js-plugin": "1.0.1",
```
Comment (aqrln) [inline packages/client/package.json:152]: somehow still in `dependencies` although the commit message says it was moved

Comment (janpio) [reply packages/client/package.json:152]: This is what it looks like when I mess up :D

**Review by aqrln — APPROVED** [review]

**Comment by aqrln** [comment]
Let's merge and try this!

**Pull request merged**